Client/mods/deathmatch/logic/CClientGame.cpp

```diff
@@ -1537,6 +1537,13 @@ void CClientGame::DoVehicleInKeyCheck()
             else
             {
                 // Enter
+                // Are we holding the aim_weapon key?	
+                SBindableGTAControl* pBind = g_pCore->GetKeyBinds()->GetBindableFromControl("aim_weapon");	
+                if (pBind && pBind->bState)	
+                {	
+                    // Stop because the player is probably doing special attack	
```

Comment:
Please remove extra tabs at the end of lines.

Reply:
Done